doc/main.html

```diff
@@ -610,15 +610,21 @@ <h4>Additional elements section</h4>
         <ul>
           <li>an <code>id</code> attribute whose value is prefixed with <code>element-</code>;</li>
           <li>a <code>title</code> attribute that provides a short description of the element;</li>
-          <li>a <code>href</code> attribute that links to the element itself, typically in the <code>doc/elements</code>
-          directory.</li>
+          <li>a <code>href</code> attribute that links to the element itself</li>
         </ul>
 
+        <p>
```

Comment:
Each additional element shall be managed in one of the following ways:

* as a file in the doc/elements directory, in which case the link to the element shall be relative path to the file, e.g. "elements/element.txt".
* in a separate SMPTE repository, in which case the link to the element shall be a versioned link to the repo, e.g. https://github.com/SMPTE/html-pub/tree/631ed7a0312fa183c356a7b742129ce46f92736f

Reply:
updated in https://github.com/SMPTE/html-pub/pull/137/commits/b72678433517467034a0ee22a61d202f3d65e3a3